tests/audio/test_actor.py

```diff
@@ -629,3 +629,54 @@ def test_change_to_stopped_while_buffering(self):
         self.audio.stop_playback()
         playbin.set_state.assert_called_with(Gst.State.NULL)
         assert not self.audio._buffering
+
+
+class AudioLiveTest(unittest.TestCase):
+    config = {
+        "audio": {
+            "buffer_time": None,
+            "mixer": "fakemixer track_max_volume=65536",
+            "mixer_track": None,
+            "mixer_volume": None,
+            "output": "testoutput",
+            "visualizer": None,
+        }
+    }
+
+    def setUp(self):  # noqa: N802
+        config = {
+            "audio": {
+                "buffer_time": None,
+                "mixer": "foomixer",
+                "mixer_volume": None,
+                "output": "testoutput",
+                "visualizer": None,
+            },
+            "proxy": {"hostname": ""},
+        }
+        self.audio = audio.Audio(config=config, mixer=None)
+
+    def test_not_live_mode(self):
+        source = mock.MagicMock()
+
+        # disable appsrc configure
```

Comment:
```suggestion
        # Avoid appsrc.configure()
```

Same in the next test.